hydra-node/test/Hydra/Chain/Direct/Contract/Close.hs

```diff
@@ -99,10 +104,12 @@ healthyCloseDatum =
     }
 
 healthyContestationPeriod :: OnChain.ContestationPeriod
-healthyContestationPeriod = OnChain.contestationPeriodFromDiffTime $ fromInteger healthyContestationPeriodSeconds
+healthyContestationPeriod = OnChain.contestationPeriodFromDiffTime $ fromInteger genContestationPeriodSeconds
 
-healthyContestationPeriodSeconds :: Integer
-healthyContestationPeriodSeconds = 10
+-- NB: We don't want to wait too long for contestation period to pass
+-- so constraining this to < 100
+genContestationPeriodSeconds :: Integer
+genContestationPeriodSeconds = arbitrary `suchThat` (< 100) `generateWith` 42
```

Comment:
These `gen...` functions are not really generators. Rename them back to indicate that they are just picked values (see comment above)